generators/upgrade/index.js

```diff
@@ -145,12 +146,26 @@ module.exports = UpgradeGenerator.extend({
 
         prepareUpgradeBranch: function() {
             var done = this.async();
+            var getGitVersion = function(callback) {
```

Comment:
I think its high time we start using arrow functions so lets start doing that.
we can avoid the doubel bind by doing
```
var getGitVersion = (callback) => {
    this.gitExec(['--version'], (code, msg)  => {
           callback(String(msg.match(/([0-9]+\.[0-9]+\.[0-9]+)/g)));
    });
};
```